src/plots/cartesian/graph_interact.js

```diff
@@ -618,7 +618,7 @@ fx.getClosest = function(cd, distfn, pointData) {
         // do this for 'closest'
         for(var i=0; i<cd.length; i++) {
             var newDistance = distfn(cd[i]);
-            if(newDistance < pointData.distance) {
+            if(newDistance <= pointData.distance) {
```

Comment:
yeah, it's a little hacky. Ideally `hoverinfo: 'none'` traces wouldn't be passed to the hover label routine, but that caused a `plotly_hover` test to fail, so I decided to go with :arrow_double_up: .